### PR TITLE
Document how to connect HTTP JSON API Service to an HA participant

### PR DESCRIPTION
**Pull request by stefanobaghino-da**
Fixes https://github.com/digital-asset/daml/issues/13369

changelog_begin
changelog_end

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [x] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description
- [ ] If you mean to change the status of a component, please make sure you keep [the Component Status page](https://github.com/digital-asset/daml/blob/main/docs/source/support/component-statuses.rst) up to date.

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

**Review by RayRoestenburg — APPROVED** [review]

**Review by ray-roestenburg-da — APPROVED** [review]

File: docs/source/json-api/production-setup.rst [hunk 1]
```diff
@@ -204,13 +204,24 @@ Users may consider running PostgreSQL backend in a `high availability configurat
...
-Finally we recommend using app orchestration utilities which monitor the health of the service
-and perform subsequent operations to ensure availability. These utilities can use the
+Finally we recommend using orchestration systems or load balancers which monitor the health of
+the service and perform subsequent operations to ensure availability. These systems can use the
 `healthcheck endpoints <https://docs.daml.com/json-api/index.html#healthcheck-endpoints>`__
 provided by the *HTTP JSON API* server. This can also be tied into supporting arbitrary
 autoscaling implementation to ensure minimum number of *HTTP JSON API* servers on
 failures.
 
+Setup the HTTP JSON API Service to work with Highly Available Participants
```
Comment (carrielaben-da) [inline docs/source/json-api/production-setup.rst:214]: The verb form is two words (set up), setup is the noun form.

File: docs/source/json-api/production-setup.rst [hunk 2]
```diff
@@ -204,13 +204,24 @@ Users may consider running PostgreSQL backend in a `high availability configurat
...
+--------------------------------------------------------------------------
+
+In case the participant node itself is configured to be highly available, depending on the setup you might want
```
Comment (carrielaben-da) [inline docs/source/json-api/production-setup.rst:217]: "If the participant node itself is configured to be highly available, you can choose different strategies [to do what?] depending on the setup."

File: docs/source/json-api/production-setup.rst [hunk 3]
```diff
@@ -204,13 +204,24 @@ Users may consider running PostgreSQL backend in a `high availability configurat
...
+to choose different strategies. In most setups, including those based on Canton, you'll likely have an active
+participant node whose role can be taken over by a passive node in case the currently active one drops. Just as
+for the *HTTP JSON API* itself, you can again use orchestration systems or load balancers to monitor the status
```
Comment (carrielaben-da) [inline docs/source/json-api/production-setup.rst:220]: can omit the word "again" here.

**Review by carrielaben-da — COMMENTED** [review]
Small grammar/usage suggestions.

File: docs/source/json-api/production-setup.rst (hunk 1 above)
Comment (stefanobaghino-da) [reply docs/source/json-api/production-setup.rst:214]: Addressed by https://github.com/digital-asset/daml/commit/508ac1e2cbb312613b1cd5004fa471e00dba523b

File: docs/source/json-api/production-setup.rst (hunk 2 above)
Comment (stefanobaghino-da) [reply docs/source/json-api/production-setup.rst:217]: Addressed by https://github.com/digital-asset/daml/commit/9fd3f064758f65b04e349a175d183236c1b8371f

File: docs/source/json-api/production-setup.rst (hunk 3 above)
Comment (stefanobaghino-da) [reply docs/source/json-api/production-setup.rst:220]: Addressed by https://github.com/digital-asset/daml/commit/291cd7bce25775f9ae1230306503b7308b762a39

**Pull request merged**